webapp/channels/src/plugins/export.js

```diff
@@ -11,7 +11,6 @@ import ChannelMembersModal from 'components/channel_members_modal';
 import {openPricingModal} from 'components/global_header/right_controls/plan_upgrade_button';
 import {useNotifyAdmin} from 'components/notify_admin_cta/notify_admin_cta';
 import PostMessagePreview from 'components/post_view/post_message_preview';
-import PurchaseModal from 'components/purchase_modal';
```

Comment:
Other plugins, maybe calls may still need this reference though

Reply:
I've checked calls, it does not use it. Playbooks has a reference to it, but, it's removal won't cause a webapp crash because it checks for existence before mounting it https://github.com/mattermost/mattermost-plugin-playbooks/blob/master/webapp/src/components/cloud_modal.tsx#L16

Reply:
Thanks for checking. 

@esethna with the Playbooks packaging changes we should revisit any Pro upsells that are still around as we no longer have a purchase modals in product and they may still be referenced in Playbooks.

Reply:
I have a PR open here to switch them to open contact sales :) https://github.com/mattermost/mattermost-plugin-playbooks/pull/1903